src/components/MapTab.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="container">
+    <map-component></map-component>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, watch, nextTick, onMounted, reactive } from 'vue';
+import MapComponent from './map/MapComponent.vue';
+</script>
+
+<style lang="scss">
+.container {
+  display: grid;
+  grid-template-columns: max-content auto;
+  grid-gap: 0.6em 3em;
+  align-items: center;
+  position: relative;
+  height: 100vh;
+}
+
+.map_container {
+  height: calc(80vh - 96px);
+  width: 100vw;
+}
+
+.map_controls {
+  position: absolute;
+  top: 25px;
+  left: 25px;
+  z-index: 2;
+}
+
+.map_search {
+  position: absolute;
+  top: 25px;
+  right: 25px;
+  z-index: 2;
+}

```

Comment:
These should be moved to `MapComponent.vue` file. Also, add `scoped` or `module` attribute to the `<style>` tag so the styles don't clash with other classes with same name outside of this component. The syntax is slightly different but you get intellisense when writing the class name in exchange. Check `App.vue`. It's also possible to use `scoped` attribute instead of `module` but I think we should stick to one of those to not make things confusing.

Reply:
Thanks, I forgot about that. Module styles look cool, but I guess they can't be combined with Vuetify utility classes, right?

Reply:
Seems like this syntax works: `:class="[$style.topBar, 'ml-5']`